explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -192,18 +193,19 @@ explorerHandlers _sendActions =
 -- API Functions
 ----------------------------------------------------------------
 
-getTotalAda :: ExplorerMode ctx m => m CCoin
+getTotalAda :: ExplorerMode ctx m => m Micro
 getTotalAda = do
     utxoSum <- EX.getUtxoSum
-    -- getUtxoSum returns Integer, but for external consumers we want to provide Coin.
-    -- However, we don't want to return meaningless values to them, so we add truncation.
-    pure $ mkCCoin $ truncateIntegerToCoin utxoSum
+    validateUtxoSum utxoSum
+    pure $ fromInteger utxoSum / 1e6
```

Comment:
I have a question. Before you returned lovelaces (the smallest unit of currency), but now you aren't. 
Isn't this a `Coin`? And what is a `Micro`? Can you explain the connection to the currency/units?

Reply:
Now literally `ada` is returned as endpoint name suggests. `Micro` allows to keep number with resolution of `10^-6`, and that perfectly fits to represent `ada`

Reply:
Wait. Why don't we call that `Ada`? :confused: